### PR TITLE
TSK-1335: removed last occurences of "HEI" prefix for history events

### PR DESCRIPTION
**Pull request by mustaphazorgati**

### CODE REVIEW

File: lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskCommentServiceImpl.java [hunk 1]
```diff
@@ -22,19 +22,14 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(TaskCommentServiceImpl.class);
 
-  private static final String NOT_AUTHORIZED =
```
Comment (gitgoodjhe) [inline lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskCommentServiceImpl.java:25]: Why did you delete these? Sonarcloud was not happy without those

**Review by gitgoodjhe — CHANGES_REQUESTED** [review]

File: common/taskana-common/src/main/java/pro/taskana/common/internal/util/IdGenerator.java [hunk 2]
```diff
@@ -5,8 +5,25 @@
...
+  public static final String ID_PREFIX_TASK = "TKI";
+  public static final String ID_PREFIX_EXT_TASK = "ETI";
+  public static final String ID_PREFIX_BUSINESS_PROCESS = "BPI";
+  public static final String ID_PREFIX_ATTACHMENT = "TAI";
+
+  public static final String ID_PREFIX_TASK_COMMENT = "TCI";
+
+  public static final String ID_PREFIX_CLASSIFICATION_HISTORY_EVENT = "CHI";
+  public static final String ID_PREFIX_WORKBASKET_HISTORY_EVENT = "WHI";
+  public static final String ID_PREFIX_TASK_HISTORY_EVENT = "HEI";
```
[the remark this answers is not in the document]
Comment (mustaphazorgati) [reply common/taskana-common/src/main/java/pro/taskana/common/internal/util/IdGenerator.java:22]: true.. fixed

File: lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskCommentServiceImpl.java (hunk 1 above)
Comment (mustaphazorgati) [reply lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskCommentServiceImpl.java:25]: "We never want String constants for Exception messages or logging statements. We prefer to duplicate those messages here because of code readability" from our guidelines

**Review by gitgoodjhe — DISMISSED** [review]
